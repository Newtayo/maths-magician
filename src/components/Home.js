import React from 'react';
import './Styles/Home.css';

const Home = () => (
  <div className="home" data-testid="home-component">
    <h2 className="homeHeader">Welcome to Our Page!</h2>
    <p className="homeText">
      Lorem ipsum dolor sit amet. Ut cupiditate suscipit qui ratione galisum aut unde galisum.
      Ab rerum sunt At placeat impedit et architecto dolores aut distinctio quia? Qui quia
      voluptas sed fugiat asperiores non voluptatum dicta. Eum impedit deleniti At quidem
      eius aut eius sequi et aspernatur rerum.

      Qui nostrum enim sit aliquid minima est laborum ratione et esse praesentium non galisum
      veritatis ut laudantium minima eum dolorum odit? Aut praesentium placeat quo amet
      blanditiis est eius similique sed nihil praesentium ut quia nostrum.

      Hic voluptatem totam hic quidem quas sit voluptatibus tempore. Sit temporibus quos
      a soluta accusantium qui dolorem error et explicabo aliquid ea nesciunt reprehenderit
      et neque delectus.
    </p>
    <p className="homeText">
      Lorem ipsum dolor sit amet. Ut cupiditate suscipit qui ratione galisum aut unde
      galisum. Ab rerum sunt At placeat impedit et architecto dolores aut distinctio
      quia? Qui quia voluptas sed fugiat asperiores non voluptatum dicta. Eum impedit
      deleniti At quidem eius aut eius sequi et aspernatur rerum.

      Qui nostrum enim sit aliquid minima est laborum ratione et esse praesentium
      non galisum veritatis ut laudantium minima eum dolorum odit? Aut praesentium
      placeat quo amet blanditiis est eius similique sed nihil praesentium ut quia nostrum.

      Hic voluptatem totam hic quidem quas sit voluptatibus tempore. Sit
      temporibus quos a soluta accusantium qui dolorem error et explicabo
      aliquid ea nesciunt reprehenderit et neque delectus.
    </p>
  </div>
);

export default Home;
