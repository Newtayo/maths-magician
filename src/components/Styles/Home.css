.home {
  display: flex;
  flex-direction: column;
}

.homeText {
  text-align: justify;
  line-height: 1.2;
  margin-top: 2rem;
}
