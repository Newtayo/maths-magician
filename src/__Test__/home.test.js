import React from 'react';
import '@testing-library/jest-dom';
import { render, fireEvent } from '@testing-library/react';

import Home from '../components/Home';

test('renders correctly', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
});

//

test('should display the number click on the calculator button to the screen', async () => {
    const { getByText } = render(<Home />)
    expect(getByText("Welcome to Our Page!")).toBeInTheDocument()
});
