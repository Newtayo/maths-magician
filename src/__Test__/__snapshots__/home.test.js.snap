// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <h2
      class="homeHeader"
    >
        Welcome to Our Page!
    </h2>
    <p
      class="homeText"
    >
      Lorem ipsum dolor sit amet. Ut cupiditate suscipit qui ratione galisum aut unde galisum. Ab rerum sunt At placeat impedit et architecto dolores aut distinctio quia? Qui quia voluptas sed fugiat asperiores non voluptatum dicta. Eum impedit deleniti At quidem eius aut eius sequi et aspernatur rerum. Qui nostrum enim sit aliquid minima est laborum ratione et esse praesentium non galisum veritatis ut laudantium minima eum dolorum odit? Aut praesentium placeat quo amet blanditiis est eius similique sed nihil praesentium ut quia nostrum. Hic voluptatem totam hic quidem quas sit voluptatibus tempore. Sit temporibus quos a soluta accusantium qui dolorem error et explicabo aliquid ea nesciunt reprehenderit et neque delectus.
    </p>
    <p
      class="homeText"
    >
      Lorem ipsum dolor sit amet. Ut cupiditate suscipit qui ratione galisum aut unde galisum. Ab rerum sunt At placeat impedit et architecto dolores aut distinctio quia? Qui quia voluptas sed fugiat asperiores non voluptatum dicta. Eum impedit deleniti At quidem eius aut eius sequi et aspernatur rerum. Qui nostrum enim sit aliquid minima est laborum ratione et esse praesentium non galisum veritatis ut laudantium minima eum dolorum odit? Aut praesentium placeat quo amet blanditiis est eius similique sed nihil praesentium ut quia nostrum. Hic voluptatem totam hic quidem quas sit voluptatibus tempore. Sit temporibus quos a soluta accusantium qui dolorem error et explicabo aliquid ea nesciunt reprehenderit et neque delectus.
    </p>
  </div>
</DocumentFragment>
`;
